qa/full-cluster-restart/src/test/java/org/elasticsearch/upgrades/FullClusterRestartIT.java

```diff
@@ -108,15 +110,29 @@ public void testSearch() throws Exception {
                     mappingsAndSettings.field("type", "text");
                     mappingsAndSettings.endObject();
                 }
+                {
+                    mappingsAndSettings.startObject("binary");
+                    mappingsAndSettings.field("type", "binary");
+                    mappingsAndSettings.field("store", "true");
+                    mappingsAndSettings.endObject();
+                }
                 mappingsAndSettings.endObject();
                 mappingsAndSettings.endObject();
                 mappingsAndSettings.endObject();
             }
             mappingsAndSettings.endObject();
             client().performRequest("PUT", "/" + index, Collections.emptyMap(),
                 new StringEntity(mappingsAndSettings.string(), ContentType.APPLICATION_JSON));
+            if (oldClusterVersion.before(VERSION_5_1_0_UNRELEASED)) {
```

Comment:
I think I'd pull this to its own test case so it doesn't get confused with the other search tests. We assert with a search but I think this is more about the alias than search in general. I dunno why I feel this way. What do you think?